frontend/src/metabase/visualizations/echarts/pie/option/constants.ts

```diff
@@ -34,7 +39,7 @@ export const TOTAL_GRAPHIC_OPTION = {
     {
       type: "text",
       cursor: "text",
-      top: 25, // TODO confirm this and other style values later
+      top: 24,
```

Comment:
Unfortunately there's no existing value to go off of here, since in `PieChart.jsx` there is no explicit position defined, the two flexbox elements naturally position themselves vertically without a gap.

ECharts graphic elements do not support this kind of auto positioning, so we just have to make our eyeball it here. The old value, 25, looked close enough, I just changed it to 24 to make sure we only use even values for spacing (odd values can create asymmetry).